libsakura/src/CMakeLists.txt

```diff
@@ -51,14 +51,26 @@ endif(LOG4CXX_FOUND AND NOT "${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
 option(SCALAR "Disable auto-vectorization by compiler" OFF)
 option(BUILD_DOC "Enable/disable Doxygen generation of Sakura API HTML documentation" ON)
 
-set(SIMD_ARCH "NATIVE" CACHE STRING "SIMD architecture: one of NATIVE SSE4 AVX AVX2" )
+set(SIMD_ARCH "NATIVE" CACHE STRING "SIMD architecture: one of NATIVE SSE4 AVX AVX2 ARM" )
+
+if(SIMD_ARCH STREQUAL "NATIVE")
```

Comment:
Could you update [SetArchFlags.cmake](https://github.com/tnakazato/sakura/blob/main/libsakura/cmake-modules/SetArchFlags.cmake) instead of L.56-66? There is similar `if` block in there, and just adding one more `elseif` block for ARM should be sufficient, I think.

Reply:
I have made the suggestion as you directed.